lib/oli/delivery/settings.ex

```diff
@@ -126,6 +130,16 @@ defmodule Oli.Delivery.Settings do
     end
   end
 
+  def check_start_date(%Combined{start_date: nil}), do: {:allowed}
+
+  def check_start_date(%Combined{start_date: start_date}) do
+    if DateTime.compare(start_date, DateTime.utc_now()) == :gt do
+      {:before_start_date}
+    else
+      {:allowed}
+    end
```

Comment:
Maybe this could be written using the short `if` statement, WDYT?

Reply:
I like short `if`s :)